lib/travis/api/v3/queries/user_builds.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  class Queries::UserBuilds < Query
+    params :id
+
+    def find
+      return [] unless params["user.id"]
```

Comment:
Instead of returning an empty array, maybe return `V3::Models::Build.none`, so AR specific calls (like scoping) can be done.

Reply:
Wow, TIL - I completely did not know about `none`!